packages/react-components/stable-review/react-components.api.md

```diff
@@ -609,6 +609,8 @@ export type MessageThreadProps = {
     onLoadPreviousChatMessages?: (messagesToLoad: number) => Promise<boolean>;
     onRenderMessage?: (messageProps: MessageProps, messageRenderer?: MessageRenderer) => JSX.Element;
     onRenderFileDownloads?: (userId: string, message: ChatMessage) => JSX.Element;
+    fileDownloadErrorMessage?: string;
```

Comment:
these props should not be part of the stable API

Reply:
Done